### DISCUSSION TITLE
회원 등급 기준 설정

### DISCUSSION THREAD

**Discussion by JunYoung-C** [TEAM-TEMPUS/jjapturi#70] [General]
우선 아래와 같이 임의로 설정해봤는데 어떤가요?
- 초기 등급 : 브론즈
- 거래 10건 이상, 평점 2.5점 이상 : 실버
- 거래 50건 이상, 평점 3.0점 이상 : 골드
- 거래 100건 이상, 평점 3.5점 이상 : 플래티넘

**Comment by wyu950** [comment]
좋은 의견인거 같습니다

**Comment by yongki1403** [comment]
평점이 5점이 최대이니 평점 기준을 조금 더 올려서 설정하면 좋을 것 같습니다.
